src/foam/u2/view/FObjectView.js

```diff
@@ -203,7 +209,6 @@ foam.CLASS({
       );
 
       if ( this.data ) { this.objectClass = dataToClass(this.data); }
-      if ( ! this.data && ! this.objectClass && this.choices.length ) this.objectClass = this.choices[0][0];
```

Comment:
Why delete this?

Reply:
@kgrgreer For my understanding, this to set objectClass select the first one in the array. I remove it  beacuse we don't want select anything by default

Reply:
@yij793 'don't want to select anything by default' - for your use case or all use cases?  
We use default selection elsewhere. 
If you need this support then it's a feature we need to add.

Reply:
@jlhughes hmm, in that case. I'll make change to make it flexible